### ISSUE TITLE
Uncaught ReferenceError: chatModule is not defined

### ISSUE THREAD

**Issue by tiliev** [IgniteUI/igniteui-docfx-template#80]
We are getting this error consistently on each page and seems to come from Google Tag Manager script. It seems it injects something similar to this:
```
<script type="text/javascript" id="">chatModule&&chatModule.initialize("sales");</script>
```
I guess it is related to the chat module in http://www.infragistics.com or something.

Error:

![error](https://user-images.githubusercontent.com/6439482/52430883-c3bda400-2b0f-11e9-8897-ce282a82a201.PNG)

Search in all of the scripts for `chatModule`:

![search](https://user-images.githubusercontent.com/6439482/52430902-ce783900-2b0f-11e9-8bf0-bfb4f2958325.PNG)

Of course if we add something hacky like this:

```
    <script>
        var chatModule = chatModule || {
            initialize: function() {}
        };
    </script>
```
we won't get error anymore.

**Comment by tiliev** [comment]
@jasonzajac, do you have any idea?

**Comment by jasonzajac** [comment]
@tiliev yes, that error is related to the chat on the main website. It's made of two parts: script file containing the script for the chatModule and GTM injects inline script onto the page.

Do you still see this issue in production? I actually changed the trigger for that tag a couple weeks back. The change should prevent it from running on Angular (sample and docs), React (samples and docs) and Indigo help. Please note: that I noticed staging didn't have the same exceptions applied, so I just did that now. If you're still seeing the console error, please let me know.

**Comment by tiliev** [comment]
@jasonzajac, I cannot see it on production and staging, but I still reproduce it locally. The code is - `GTM-WLXLBZD`.

**Comment by jasonzajac** [comment]
@tiliev I didn't even know they were added into Dev. Regardless, I added the same exceptions to those environments as well. So, you shouldn't be seeing those anymore. :)

**Comment by tiliev** [comment]
@jasonzajac, I tested and I can confirm it is working without error for JP, `GTM-NNHVMC7`, but I still see the error with GTM code `GTM-WLXLBZD` for EN and KR.

**Comment by jasonzajac** [comment]
@tiliev sorry about that. so the exception applied to the tag was conditionally injecting the tag based off of Page URL -- for angular it was matching based off of regex, **ignite-ui-angular/(angular|docs).***

This isn't going to work when working locally. Since there's really no point in having it on dev anymore, I just removed the "All Page" trigger. I confirmed that it should no longer appear locally.
